features/parallel_custom_assign.feature

```diff
@@ -4,99 +4,70 @@ Feature: Running scenarios in parallel with custom assignment
   Scenario: Bad parallel assignment helper uses 1 worker
     Given a file named "features/step_definitions/cucumber_steps.js" with:
       """
-      const {Then, setParallelCanAssign} = require('@cucumber/cucumber')
+      const {Given, setParallelCanAssign} = require('@cucumber/cucumber')
+
       setParallelCanAssign(() => false)
-      Then(/^value is (\d+)$/, function(v, cb) {
-         setTimeout(cb, 150)
-      })
+
+      Given('slow step', (done) => setTimeout(done, 100))
       """
     And a file named "features/a.feature" with:
       """
       Feature: only one worker works
         Scenario: someone must do work
-          Then value is 1
+          Given slow step
 
         Scenario: even if it's all the work
-          Then value is 2
+          Given slow step
       """
     When I run cucumber-js with `--parallel 2`
     Then the error output contains the text:
     """
     WARNING: All workers went idle 2 time(s). Consider revising handler passed to setParallelCanAssign.
     """
-    And no tests ran in tandem
-
-  Scenario: Both works run tests when a valid assignment helper is used
```

Comment:
Didn't think this feature was needed given its just using the default the parallelCanAssign function (ie just standard parallel functionality)